ci-operator/step-registry/openshift/e2e/aws/mirror/openshift-e2e-aws-mirror-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: openshift-e2e-aws-mirror
+  steps:
+    pre:
+    - chain: ipi-conf-aws
+    - ref: ipi-conf-mirror
+    - chain: ipi-install
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: gather-core-dump
+    - chain: ipi-aws-post
+    dependencies:
+      RELEASE_IMAGE_LATEST: "release:mirrored"
```

Comment:
I'm not clear on how this is working.  I would expect the workflow to be something we could drop in to generic installs/updates without needing to add a new `dependency` image.